airbyte-integrations/connectors/source-apify-dataset/integration_tests/configured_catalog.json

```diff
@@ -2,7 +2,7 @@
   "streams": [
     {
       "stream": {
-        "name": "datasets",
+        "name": "Dataset Collection",
```

Comment:
@vdusek Streams gets mapped to tables in most destinations - spaces etc. get normalized but it keeps things reliable if they have a "table"-y name already.

What do you think about naming them `dataset_collection`, `dataset` and `item_collection` and explaining what they mean in the documentation

Reply:
Makes sense, updated, thanks.